facdb/sql/nysopwdd_providers.sql

```diff
@@ -0,0 +1,36 @@
+DROP TABLE IF EXISTS _nysopwdd_providers;
+
+SELECT
+    uid,
+    source,
+    initcap(service_provider_agency) as facname,
+    parsed_hnum as addressnum,
+    parsed_sname as streetname,
+    cleaned_address || ' ' || street_address_line_2 as address,
```

Comment:
This will create an extra space after the vast majority of addresses, since line 2 addresses are rare. I would either use a case statement or not include the second line.

Reply:
Also, use uncleaned version.